### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by alluxxtech**
Homework 2

### CODE REVIEW

File: logs/error.log [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (vkovallohika) [inline logs/error.log:1]: log files should be added to .gitignore

File: src/config.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+var argv = process.argv.slice(2);
+process.env.PORT = '5000';
```
Comment (vkovallohika) [inline src/config.ts]: it rewrite any value you will set as environment variable. Better to write something like 
```
Number(process.env.PORT) || 5000
```

File: src/services/Movie.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { IncomingMessage, ServerResponse } from "http";
+import { logger } from "../logger";
+
+const Movie = (req: IncomingMessage, res: ServerResponse, body: string) => {
```
Comment (vkovallohika) [inline src/services/Movie.ts:4]: Is that for homework 2 or for 3?

File: dist/logger.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+"use strict";
```
Comment (vkovallohika) [inline dist/logger.js:1]: it is something very overengineered)

**Review by vkovallohika — CHANGES_REQUESTED** [review]

File: src/config.ts (hunk 2 above)
Comment (alluxxtech) [reply src/config.ts]: okay, ths

File: logs/error.log (hunk 1 above)
Comment (alluxxtech) [reply logs/error.log:1]: added logs and dist folder

File: src/services/Movie.ts (hunk 3 above)
Comment (alluxxtech) [reply src/services/Movie.ts:4]: its homework-2

File: dist/logger.js (hunk 4 above)
Comment (alluxxtech) [reply dist/logger.js:1]: agree), added to .gitignore

**Review by vkovallohika — APPROVED** [review]

**Pull request merged**